src/Microsoft.AspNet.Mvc.Core/ActionResults/CreatedAtActionResult.cs

```diff
@@ -4,6 +4,7 @@
 using System;
 using System.Collections.Generic;
 using Microsoft.AspNet.Mvc.Core;
+using Microsoft.AspNet.WebUtilities;
```

Comment:
@Tratcher would you guys consider moving this to more common namespace? As is it kinda makes me sad :cry: